src/lib.rs

```diff
@@ -956,6 +940,7 @@ mod tests {
             Uuid::new_v3(&Uuid::NAMESPACE_DNS, "rust-lang.org".as_bytes());
 
         assert_eq!(uuid.get_version().unwrap(), Version::Md5);
+        println!("pass1");
```

Comment:
We should remove this before merging

Reply:
um... yea... I made a mistake during the refactor and couldn't figure out where was fail happening (cuz there are multiple asserts)

doing it rn